crates/env/src/api.rs

```diff
@@ -701,6 +701,106 @@ pub fn set_code_hash(code_hash: &[u8; 32]) -> Result<()> {
     <EnvInstance as OnInstance>::on_instance(|instance| instance.set_code_hash(code_hash))
 }
 
+/// **New version of this function which will replace [`set_code_hash`] in `v5.0`**
```

Comment:
How not to forget removing this command?

Reply:
Probably best to create a follow-up ticket, can also add the Hernando wishlist there:

> I like having `Hash` instead of `[u8; 32]`, but changing the `env/api` function signature would require a `MAJOR` version bump.

Reply:
I'll create a board for 5.0 soon, will then put the follow-up there.